tests/models/whisper/test_modeling_whisper.py

```diff
@@ -1912,6 +1912,177 @@ def test_tiny_timestamp_generation(self):
+
+        input_speech = np.concatenate(self._load_datasamples(4))
+        input_features = processor(
+            input_speech, return_tensors="pt", sampling_rate=16_000, return_token_timestamps=True
+        ).input_features
+        input_features = input_features.to(torch_device)
+
+        generated_ids = model.generate(input_features, max_length=448, return_timestamps=True).to("cpu")
+
+        EXPECTED_OUTPUT = torch.tensor(
+            [
+                50258,
+                50259,
+                50360,
+                50365,
+                2221,
+                13,
+                2326,
+                388,
+                391,
+                307,
+                264,
+                50244,
+                295,
+                264,
+                2808,
+                5359,
+                11,
+                293,
+                321,
+                366,
+                5404,
+                281,
+                2928,
+                702,
+                14943,
+                13,
+                50629,
+                50682,
+                6966,
+                307,
+                2221,
+                13,
+                2326,
+                388,
+                391,
+                311,
+                9060,
+                1570,
+                1880,
+                813,
+                702,
+                1871,
+                13,
+                50870,
+                50911,
+                634,
+                5112,
+                505,
+                300,
+                412,
+                341,
+                42729,
+                3196,
+                295,
+                264,
+                1064,
+                11,
+                365,
+                5272,
+                293,
+                12904,
+                9256,
+                450,
+                10539,
+                949,
+                505,
+                11,
+                51245,
+                51287,
+                1034,
+                4680,
+                10117,
+                490,
+                3936,
+                293,
+                1080,
+                3542,
+                5160,
+                881,
+                26336,
+                281,
+                264,
+                1575,
+                13,
+                51494,
+                51523,
+                634,
+                575,
+                12525,
+                22618,
+                1968,
+                6144,
+                35617,
+                1456,
+                397,
+                266,
+                311,
+                589,
+                307,
+                534,
+                10281,
+                934,
+                439,
+                11,
+                51799,
+                51815,
+                50257,
+            ]
+        )
+        self.assertTrue(torch.allclose(generated_ids, EXPECTED_OUTPUT))
```

Comment:
`.allclose` doesn't really make sense for ints?

```suggestion
        self.assertTrue((generated_ids == EXPECTED_OUTPUT).all())
```